server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1264,7 +1269,12 @@ public IStorageGroupMNode getStorageGroupNodeByPath(PartialPath path) throws Met
 
   /** Get all storage group MNodes */
   public List<IStorageGroupMNode> getAllStorageGroupNodes() {
-    return mtree.getAllStorageGroupNodes();
+    try {
+      return mtree.getAllStorageGroupNodes();
+    } catch (MetadataException e) {
+      logger.warn("Something wrong when collecting storage groups");
```

Comment:
This should be an error rather than warn.

Reply:
Fixed.